src/components/Header.jsx

```diff
@@ -1,44 +1,49 @@
-import {Link} from "react-router-dom";
+import { Link } from "react-router-dom";
 import "../App.css";
+import { useCart } from "../context/CartContext";
 
 const Header = () => {
-    return(
-        <nav className="navigation nav-bar flex-row">
-        <div className="p-1">
-          <Link to="/" className="navbar-brand brand-heading font-lg" >Gourmet</Link>
-        </div>
-        <div className="flex-row">
-          <div className="cart-input flex-row">
-            <input type="text" placeholder="search" />
-            <Link to="/" className="btn-link-noBorder">   
-              <span className="material-icons">search</span>
+  const { cartState } = useCart();
+  const { cartBasket } = cartState;
+  return (
+    <nav className="navigation nav-bar flex-row">
+      <div className="p-1">
+        <Link to="/" className="navbar-brand brand-heading font-lg">
+          Gourmet
+        </Link>
+      </div>
+      <div className="flex-row">
```

Comment:
Try to use semantic tags often instead of divs

Reply:
will keep that in mind going forward